test/e2e/storage/testsuites/testdriver.go

```diff
@@ -141,6 +140,7 @@ type DriverInfo struct {
 	FeatureTag       string // FeatureTag for the driver
 
 	MaxFileSize          int64               // Max file size to be tested for this driver
+	SupportedSizeRange   volume.SizeRange    // The range of size supported by this driver
```

Comment:
Does this have to be in DriverInfo?

Reply:
I think it is something like `SupportedFsType` and `SupportedMountOption`, So I add it to DriverInfo